src/main/java/ch/wisv/areafiftylan/extras/consumption/model/ConsumptionMap.java

```diff
@@ -37,7 +38,7 @@
 
     @NonNull
     @ManyToMany(fetch = FetchType.EAGER)
-    private Collection<Consumption> consumptionsMade;
+    private List<Consumption> consumptionsMade;
```

Comment:
In my opinion this is ok, an more abstract class allows for more kinds of internal return types, allowing for more optimization. This will not change the data returned, it will still be an array-like return.